Cargo.toml

```
@@ -2,21 +2,20 @@
 name = "dazeus-plugin-karma"
 version = "0.1.0"
 authors = ["Ruben Nijveld <ruben@gewooniets.nl>"]
```

```suggestion
authors = ["Ruben Nijveld <ruben@gewooniets.nl>", "Daan Sprenkels"]
```
By this point (fix for email)